### PR TITLE
CI job for testing kubetest2-ec2 against k/k

### PR DESCRIPTION
**Pull request by dims**
`kubetest2-ec2` is a new `kubetest2` deployer that starts a ec2 node with kubeadm and runs tests against it.

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/ec2-containerd.yaml [hunk 1]
```diff
@@ -12,6 +12,67 @@ presets:
...
+           - |
+             go install sigs.k8s.io/provider-aws-test-infra/kubetest2-ec2@latest
+             kubetest2 ec2 \
+              --build \
+              --instance-type=m6a.large  \
+              --images=ami-02675d30b814d1daa \
+              --region us-east-1 \
+              --target-build-arch linux/amd64 \
+              --stage provider-aws-test-infra \
+              --up \
```
Comment (upodroid) [inline config/jobs/kubernetes/sig-node/ec2-containerd.yaml:55]: Where is the `--down` flag?

Comment (dims) [reply config/jobs/kubernetes/sig-node/ec2-containerd.yaml:55]: whoops. adding

**Comment by dims** [comment]
thanks @upodroid done. once i get the job working i will move the definition to a better spot, it doesn't really belong in sig-node

File: config/jobs/kubernetes/sig-node/ec2-containerd.yaml [hunk 2]
```diff
@@ -12,6 +12,68 @@ presets:
...
+              --down \
+              --user-data-file=./../../sigs.k8s.io/provider-aws-test-infra/kubetest2-ec2/config/ubuntu2204.yaml \
```
Comment (upodroid) [inline config/jobs/kubernetes/sig-node/ec2-containerd.yaml:57]: nit
```suggestion
              --user-data-file=${GOPATH}/src/sigs.k8s.io/provider-aws-test-infra/kubetest2-ec2/config/ubuntu2204.yaml \
```

**Review by upodroid — APPROVED** [review]
LGTM

**Pull request merged**